src/Athens-Examples/AthensSurfaceExamples.class.st

```diff
@@ -521,7 +521,8 @@ AthensSurfaceExamples class >> exampleClip [
 			can drawShape: (0@0 corner: 100@ 100).
 		]
 	].
-	Display getCanvas drawImage: surf asForm at: 0@0
+
+	surf asForm asMorph openInWindow
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?